airbyte-integrations/bases/connector-acceptance-test/README.md

```diff
@@ -48,31 +36,29 @@ To do so, from the root of the `airbyte` repo, run `./airbyte-cdk/python/bin/run
 * When running local acceptance tests on connector:
   * When running `connectorAcceptanceTest` `gradle` task
   * When running or `./acceptance-test-docker.sh` in a connector project
-* When running `/test` command on a GitHub pull request.
-* When running ` integration-test` GitHub action. This is the same action that creates and uploads the test report JSON files that power the badges in the [connector registry summary report](https://connectors.airbyte.com/files/generated_reports/connector_registry_report.html).
+* In the CI on each push to a connector PR.
 
 ## Developing on the acceptance tests
 You may want to iterate on the acceptance test project itself: adding new tests, fixing a bug etc.
 These iterations are more conveniently achieved by remaining in the current directory.
 
-1. Create a `virtualenv`: `python -m venv .venv`
-2. Activate the `virtualenv`: `source ./.venv/bin/activate`
-3. Install requirements: `pip install -e .`
-4. Run the unit tests on the acceptance tests themselves: `python -m pytest unit_tests` (add the `--pdb` option if you want to enable the debugger on test failure)
-5. Make the changes you want:
+1. `poetry install`
+2. Activate the `virtualenv`: `poetry shell`
```

Comment:
nit: Lets debate the poetry run vs poetry shell!
https://python-poetry.org/docs/basic-usage/

Im currently team `poetry run pytest` for three not important reasons
1. You dont have to think "Did I activate my shell?" Because it all happens in one
2. We dont trick new devs into thinking `poetry shell + pip install` is the same as `poetry add`
3. It feels closer to npm

The third item has me thinking this might just be a preference and that shell could be better for a reason I dont know about

Reply:
My team is yours. e25defe36eeccb97b8dc345e5056319a2b3b8fd6